test/ui-test/testSuites/suite_messaging/tst_chatsNavigation/test.feature

```diff
@@ -0,0 +1,34 @@
+Feature: Status Desktop Chat Navigation
+
+    As a user I want to join seethe application reflect correctly
+    when I navigate trough chats list.
+
+    The feature start sequence is the following (setup on its own `bdd_hooks`):
+    ** given A first time user lands on the status desktop and generates new key
+    ** when user signs up with username "tester123" and password "TesTEr16843/!@00"
+    ** and the user lands on the signed in app
+
+	[Cleanup] Also each scenario starts with:
+	** when the user opens the chat section
+
+    Scenario: The user joins a room and marks it as read
+        When the user joins chat room "test"
+        And the user marks the channel "test" as read
+        # TODO find a way to validate that it worked
+
+    Scenario: The user sees chats sorted by most recent activity
+        When the user joins chat room "first-chat"
+        And the user joins chat room "second-chat"
+        And the user joins chat room "third-chat"
+        Then the user chats are sorted accordingly
+        | third-chat  |
+        | second-chat |
+        | first-chat  |
+        When the user switches to "second-chat" chat
```

Comment:
Is switches to and joins different behaviour, or different name for same thing?

Reply:
OK, fine to leave this as it is then. Ideally we can find some terminology to make this distinction clearer in future.